File: tests/expected/cls.dart
// @dart=3.4
import 'package:sprintf/sprintf.dart';

class Foo {
  String bar() {
    return "a";
  }
}

main(List<String> argv) {
  final Foo f = Foo();
  final b = f.bar();
  print(sprintf("%s", [b]));
}

File: tests/expected/cls.cpp
#include <cstdint>   // NOLINT(build/include_order)
#include <iostream>  // NOLINT(build/include_order)
class Foo {
 public:
  inline std::string bar() { return std::string{"a"}; }
};

int main(int argc, char** argv) {
  Foo f = Foo();
  auto b = f.bar();
  std::cout << b;
  std::cout << std::endl;
}
